core/src/test/java/hudson/util/ArgumentListBuilderTest.java

```diff
@@ -101,28 +104,51 @@ public void assertPrependBeforeAddingMasked() {
 
     @Test
     public void testToWindowsCommand() {
-        ArgumentListBuilder builder = new ArgumentListBuilder(
-                "ant.bat", "-Dfoo1=abc",  // nothing special, no quotes
-                "-Dfoo2=foo bar", "-Dfoo3=/u*r", "-Dfoo4=/us?",  // add quotes
-                                                 "-Dfoo10=bar,baz",
-                "-Dfoo5=foo;bar^baz", "-Dfoo6=<xml>&here;</xml>", // add quotes
-                "-Dfoo7=foo|bar\"baz", // add quotes and "" for "
-                "-Dfoo8=% %QED% %comspec% %-%(%.%", // add quotes, and extra quotes for %Q and %c
-                "-Dfoo9=%'''%%@%"); // no quotes as none of the % are followed by a letter
+    	ArgumentListBuilder builder = new ArgumentListBuilder().
+        		add("ant.bat").add("-Dfoo1=abc").  // nothing special, no quotes
```

Comment:
please be consistent with the use of tabs and spaces within the same file and certainly the same function.